.github/workflows/release.yml

```diff
@@ -0,0 +1,71 @@
+name: 'release'
+on:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  # .goreleaser.docker.yaml reads CONTAINER_REGISTRY
+  CONTAINER_REGISTRY: 'us-docker.pkg.dev/abcxyz-artifacts/docker-images'
```

Comment:
Does the SA has writer permission to it?

Reply:
Not yet, will grand the sa right permission in infra in next pr